cpp/include/cudf/groupby.hpp

```diff
@@ -19,7 +19,9 @@
 #include <cudf/aggregation.hpp>
 #include <cudf/table/table_view.hpp>
 #include <cudf/types.hpp>
+#include <rmm/device_buffer.hpp>
 
+#include <memory>
```

Comment:
Were these needed / do we need a cpp review for this?

Reply:
Got rid of them and nothing broke